lib/ansible/modules/apt.py

```diff
@@ -1257,7 +1258,7 @@ def main():
         #    made any more complex than it already is to try and cover more, eg, custom interpreters taking over
         #    system locations)
 
-        apt_pkg_name = 'python3-apt' if PY3 else 'python-apt'
```

Comment:
We might want to make this change in diff PR as this breaks existing 2.7 clients (which we break elsewhere i know, but people can still copy back the apt module itself to older ansible).